middleware/validation.js

```diff
@@ -0,0 +1,85 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const bahmniInputValidations = require('../endpoints/bahmni/input-validation.json')
+const openIMISInputValidations = require('../endpoints/openIMIS/input-validation.json')
+
+// This method creates a joi vaidation schema. It returns null if the scheam creation fails
+const createJoiValidationSchema = (resourceName) => {
+  let validations
+  const schemaObject = {}
+
+  // TODO add more cases for other resources
+  switch (resourceName) {
+    case 'bahmniPatient':
```

Comment:
@bradsawadye how are you going to handle other types of schemas without changing this code?

Reply:
Fixed this in  https://github.com/jembi/openhim-mediator-mapping/pull/6/commits/db9274bc7756c418f269f9df16c241ad9ac60307